tests/test-order.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+/**
+ * @package Zota_Woocommerce
+ */
+
+/**
+ * Class WC_Tests_Payment_Gateway.
+ */
+class WC_Tests_Order extends WC_Unit_Test_Case {
+
+	/**
+     * Data Array
+     * @return array
+     */
+    public function getDataPartialPayment()
+    {
+        $stream = dirname(__FILE__) . '/data/callback-partial-payment.json';
+        $fileContents = \file_get_contents($stream);
+        $data = \json_decode($fileContents, JSON_OBJECT_AS_ARRAY);
+
+        return [
+            [
+                [
+                    'stream' => $stream,
+                    'data' => $data,
+                ],
+            ],
+        ];
+    }
```

Comment:
The indentation here is off, convert it to tabs maybe?

Reply:
I don't understand that.. There are tabs..